### PR TITLE
feat: Add template for Github Codespaces

### PR DESCRIPTION
**Pull request by sfc-gh-madkins**
**What this PR does / why we need it**:

Make it easier for people to develop feast

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by sfc-gh-madkins** [comment]
@adchia this is actually a very slow way to get someone started with github codespaces, we should just publish a docker image that can be pulled in to reduce startup times.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      "version": "latest"
+    },
+    "ghcr.io/devcontainers/features/python:1": {
+      "version": "3.9"
+    },
+    "ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {
+      "version": "latest"
+    }
```
Comment (achals) [inline .devcontainer/devcontainer.json:13]: What does this layer do?

**Review by achals — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (sfc-gh-madkins) [reply .devcontainer/devcontainer.json:13]: It installs brew, so I can easily install mysql :)

**Pull request merged**